lib/runtime/application.ts

```diff
@@ -197,11 +200,18 @@ export default class Application extends Addon {
   private async createServer(port: number): Promise<void> {
     await new Promise((resolve) => {
       let handler = this.router.handle.bind(this.router);
+      let server: any;
       if (this.config.server.ssl) {
-        https.createServer(this.config.server.ssl, handler).listen(port, resolve);
+        server = https.createServer(this.config.server.ssl, handler).listen(port, resolve);
       } else {
-        http.createServer(handler).listen(port, resolve);
+        server = http.createServer(handler).listen(port, resolve);
       }
+      this.drainers.push(async function drainHttp() {
+        await new Promise((resolve) => {
```

Comment:
There's still the potentially confusing naming similarity here, but as you said before, it shouldn't break anything necessarily.